doc/source/getting-started/qibosoq_lab_cal.rst

```diff
@@ -0,0 +1,486 @@
+      controller = RFSoC(NAME, ADDRESS, PORT)
+
+      # Create channel objects and port assignment
+      channels = ChannelMap()
+      channels |= Channel("readout", port=controller[1])  # QICK DAC number
+      channels |= Channel("feedback", port=controller[0])  # QICK ADC number
+      channels |= Channel("drive", port=controller[0])  # QICK DAC number
+
+      # create qubit objects
+      runcard = load_runcard(runcard_path)
+      qubits, pairs = load_qubits(runcard)
+      # assign channels to qubits
+      qubits[0].readout = channels["readout"]
+      qubits[0].feedback = channels["feedback"]
+      qubits[0].drive = channels["drive"]
+
+      instruments = {controller.name: controller}
+      settings = load_settings(runcard)
+      return Platform(NAME, qubits, pairs, instruments, settings, resonator_type="3D")
+
+
+File ``platform.yml``:
+
+.. code-block:: yaml
+
+  nqubits: 1
+  qubits: [0]
+  topology: []
+  settings: {nshots: 1024, relaxation_time: 70000, sampling_rate: 9830400000}
+
+  native_gates:
+      single_qubit:
+          0:
+              RX:  # pi-pulse for X gate
+                  duration: 40
+                  amplitude: 0.5
+                  frequency: 5_500_000_000
+                  shape: Gaussian(3)
+                  type: qd
+                  start: 0
+                  phase: 0
+
+              MZ:  # measurement pulse
+                  duration: 2000
+                  amplitude: 0.02
+                  frequency: 7_370_000_000
+                  shape: Rectangular()
+                  type: ro
+                  start: 0
+                  phase: 0
+
+      two_qubits: {}
+  characterization:
+      single_qubit:
+          0:
+              readout_frequency: 7370000000
+              drive_frequency: 5500000000
+              anharmonicity: 0
+              Ec: 0
+              Ej: 0
+              g: 0
+              T1: 0.0
+              T2: 0.0
+              threshold: 0.0
+              iq_angle: 0.0
+              mean_gnd_states: [0.0, 0.0]
+              mean_exc_states: [0.0, 0.0]
+
+
+Every experiment, will then start with:
+
+.. code-block:: python
+
+  from qibolab import create_platform
+  from qibolab import AcquisitionType, AveragingMode, ExecutionParameters
+
+  from qibolab.pulses import (
+      DrivePulse,
+      ReadoutPulse,
+      PulseSequence,
+  )
+
+  # Define platform and load specific runcard
+  platform = create_platform("platform")
+
+Qibocal
+-------
+
+For Qibocal, we first need to setup Qibolab as presented in the last section.
+Note then that, for Qibocal "programs" we need a new file ``actions.yml`` that will contain all the parameters required for the experiments: this file will be presented for all the different experiments.
+For Qibosoq and Qibolab "programs", a standard Python script or a Jupyter Notebook will suffice.
+
+
+Time Of Flight
+""""""""""""""
+
+Qibosoq
+-------
+
+.. code-block:: python
+
+  from qibosoq.components.pulses import Rectangular
```

Comment:
Even though you said it above in the preparation, it might be worth to repeat the setup over and over, to make the scripts (i.e. code blocks) self-contained (I'm pretty sure the average user will copy-paste, and then wonder why it is not working...).